groups/createElements.js

```diff
@@ -123,28 +123,40 @@ const createGroupCreationModal = (onClose = () => {}, onSubmit = () => {}) => {
                 </button>
               </div>
             </div>
+            <div class="input" id="description">
+              <label for="description" class="input__label">Description</label>
+              <div class="input__container">
+                <textarea
+                  class="input__field"
+                  name="description"
+                  placeholder="This group is for..." 
+                ></textarea>
+              </div>
+            </div>
 
             <div class="spacer"></div>
             <button type="submit" class="submit__button submit__button--disabled button">Create</button>
         </form>
     `;
-  const inputField = modalElement.querySelector('.input__field');
+  const titleField = modalElement.querySelector('#title .input__field');
+  const descriptionField = modalElement.querySelector(
+    '#description .input__field',
+  );
   const submitButton = modalElement.querySelector('.submit__button');
   modalElement.onclick = (e) => e.stopPropagation();
   modalElement.querySelector('#close-button').onclick = onClose;
   modalElement.querySelector('#clear-input').onclick = () => {
-    inputField.value = '';
+    titleField.value = '';
     toggle({ enabled: false });
   };
 
-  function getInput() {
-    return inputField.value.trim().toLowerCase();
+  function getTitle() {
+    return titleField.value.trim().toLowerCase();
   }
 
-  inputField.addEventListener('input', () => {
-    const inputValue = getInput();
-    const shouldDisableSubmit =
-      inputValue.length === 0 || inputValue.includes('group');
+  titleField.addEventListener('input', () => {
+    const title = getTitle();
+    const shouldDisableSubmit = title.length === 0 || title.includes('group');
```

Comment:
You can also consider optimising event listeners and functions. Combine similar event listeners or break down large functions into smaller, more manageable ones.